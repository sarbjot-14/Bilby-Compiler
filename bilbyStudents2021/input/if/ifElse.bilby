main{
	if(false){
	print "if failed false" $n;
	}
	else{
	print "else worked" $n;
	}
	
	if(true){
	print "if passed true" $n;
	}
	else{
	print "else failed" $n;
	}
	
	if(2!=2){
	print "if failed false" $n;
	}
	else{
		print "else passed " $n;
		if(true){
			print "else if passed" $n;
		}
	}

}