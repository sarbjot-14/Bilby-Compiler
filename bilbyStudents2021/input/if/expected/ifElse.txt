else worked
if passed true
else passed 
else if passed
